src/lib/scanner-strategies/v5-scanner.ts

```diff
@@ -28,52 +33,57 @@ type RuleDescription = {
 }
 
 export class CliScannerV5Strategy extends CliScannerStrategy {
-    private readonly name: string = '@salesforce/plugin-code-analyzer@^5 via CLI';
     private readonly settingsManager: SettingsManager;
-    private installedCodeAnalyzerVersion?: semver.SemVer;
+    private readonly display: Display;
+
+    private installedCliVersion?: semver.SemVer;
     private ruleDescriptionMap?: Map<string, string>;
 
-    public constructor(settingsManager: SettingsManager) {
+    public constructor(settingsManager: SettingsManager, display: Display) {
         super();
         this.settingsManager = settingsManager;
+        this.display = display;
     }
 
     public override getScannerName(): string {
-        return this.name;
+        return '@salesforce/plugin-code-analyzer@^5 via CLI';
     }
 
     protected override async validatePlugin(): Promise<void> {
-        const absoluteMinVersion: semver.SemVer = new semver.SemVer(ABSOLUTE_MINIMUM_REQUIRED_CODE_ANALYZER_CLI_PLUGIN_VERSION);
+        const absMinVersion: semver.SemVer = new semver.SemVer(ABSOLUTE_MINIMUM_REQUIRED_CODE_ANALYZER_CLI_PLUGIN_VERSION);
         const recommendedMinVersion: semver.SemVer = new semver.SemVer(RECOMMENDED_MINIMUM_REQUIRED_CODE_ANALYZER_CLI_PLUGIN_VERSION);
 
-        if (!this.installedCodeAnalyzerVersion) {
-            this.installedCodeAnalyzerVersion = await getCodeAnalyzerCliVersion();
-        }
-
-        if (!this.installedCodeAnalyzerVersion) {
-            throw new Error(messages.error.codeAnalyzerMissing);
-        } else if (semver.lt(this.installedCodeAnalyzerVersion, absoluteMinVersion)) {
-            throw new Error(
-                messages.error.codeAnalyzerDoesNotMeetMinVersion(
-                    this.installedCodeAnalyzerVersion.toString(),
-                    absoluteMinVersion.toString()
-                ) + '\n' + messages.error.codeAnalyzerMissing);
-        } else if (semver.lt(this.installedCodeAnalyzerVersion, recommendedMinVersion)) {
-            // TODO: Need to somehow throw warning
+        const installedCliVersion: semver.SemVer | undefined = await this.getInstalledCliVersion();
```

Comment:
Same ask for this - could this move to a subsequent PR for validating the version/environment?